api/tests/test_opening_hour_api.py

```diff
@@ -0,0 +1,41 @@
+import datetime
+
+import pytest
+from assertpy import assert_that
+from rest_framework.reverse import reverse
+
+from opening_hours.hours import TimeElement
+
+
+@pytest.mark.django_db
+def test_fetching_opening_hours_for_reservation_unit(
+    user_api_client, reservation_unit, mocker
+):
+    mocker.patch(
```

Comment:
you could use the unittest mock and add a decorator; `@mock.patch("api.hauki_api.get_opening_hourse", return_value=something)`